src/main/java/org/bitsofinfo/hazelcast/discovery/docker/swarm/DockerSwarmDiscoveryStrategyFactory.java

```diff
@@ -36,7 +36,7 @@ public DiscoveryStrategy newDiscoveryStrategy(DiscoveryNode discoveryNode,
                                                   ILogger logger,
                                                   Map<String, Comparable> properties) {
 
-        return new DockerSwarmDiscoveryStrategy(discoveryNode, logger, properties);
```

Comment:
Do not alter/remove any public signatures/constructors. This could break others using this library in use-cases specific to them such as programmatically configuring the stack.

Reply:
The passed ILogger is never used.  Surely best to clean that up with an appropriate release note?  Happy to revert if you wish....

Reply:
Please remove it